acceptance/tests/provider/package/yum.rb

```diff
@@ -0,0 +1,104 @@
+test_name "yum provider should install, uninstall, upgrade, downgrade, and use semantic versioning for ensuring desired version" do
```

Comment:
Why are we not using the tests that are already in: https://github.com/puppetlabs/puppet/blob/master/acceptance/tests/resource/package/yum.rb

Seems confusing to have acceptance tests for yum in 2 places

Reply:
didn't knew we had that file there. Will merge the semantic version test there

Reply:
Or better to just modify my acceptance to test semantic versioning

Reply:
Will not run since that file is set to audit:medium, maybe we have a separate file for testing the range, with a single test that install a version between 2 boundaries. Acceptance test are expensive and if you can cover some scenarios in unit tests, this will win time for us.